src/data/zwave_js.ts

```diff
@@ -58,6 +86,38 @@ export const fetchNodeStatus = (
     node_id,
   });
 
+export const fetchNodeConfigParameters = (
+  hass: HomeAssistant,
+  entry_id: string,
+  node_id: number
+): Promise<ZWaveJSNodeConfigParams[]> =>
+  hass.callWS({
+    type: "zwave_js/get_config_parameters",
+    entry_id,
+    node_id,
+  });
+
+export const setNodeConfigParameter = (
+  hass: HomeAssistant,
+  entry_id: string,
+  node_id: number,
+  property: number,
+  value: number,
+  property_key: number | null
+): void => {
+  const data: ZWaveJSSetConfigParamData = {
+    type: "zwave_js/set_config_parameter",
+    entry_id: entry_id,
+    node_id: node_id,
+    property: property,
+    value: value,
+  };
+  if (property_key !== null) {
+    data.property_key = property_key;
+  }
+  hass.callWS(data);
```

Comment:
Return this promise.

That way the caller can a) check to ensure the backend has received it and b) handle errors if it doesn't.